File: src/main/java/xyz/theprogramsrc/simplecoreapi/bungee/BungeeLoader.java
package xyz.theprogramsrc.simplecoreapi.bungee;

import net.md_5.bungee.api.plugin.Plugin;
import xyz.theprogramsrc.simplecoreapi.global.SimpleCoreAPI;

public class BungeeLoader extends Plugin {
 
    @Override public void onEnable(){
        new SimpleCoreAPI();
    }
    
}

File: src/main/kotlin/xyz/theprogramsrc/simplecoreapi/bungee/BungeeLoader.kt
package xyz.theprogramsrc.simplecoreapi.bungee

import net.md_5.bungee.api.plugin.Plugin
import xyz.theprogramsrc.simplecoreapi.global.SimpleCoreAPI
import xyz.theprogramsrc.simplecoreapi.global.utils.logger.JavaLogger

class BungeeLoader: Plugin() {

    companion object {
        lateinit var instance: BungeeLoader
            private set
    }

    override fun onLoad() {
        instance = this
        SimpleCoreAPI(JavaLogger(this.logger))
    }

}